setup.py

```diff
@@ -37,13 +37,13 @@
     keywords="simple climate model",
     license="GNU Affero General Public License v3.0 or later",
     packages=["openscm"],
-    tests_require=["pytest"],
+    install_requires=["pyam-iamc"],
```

Comment:
shoulg'nt that be `pint`?

Reply:
We need `pyam-iamc` for our high level dataframe.

It will be Pint, once the new release is done (or given we're not using units in dataframes yet we might be able to do it now.., I'll check in another PR).